### PR TITLE
cmd/utils, core/rawdb, triedb/pathdb: flip hash to path scheme

### PR DESCRIPTION
**Pull request by sysvm**
### Description

This PR flips the default state-scheme from hash to path, though it also adds a fallback for archive mode which force sets hash mode.

### Rationale

N/A

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

**Review by zzzckck — DISMISSED** [review]
LGTM

**Review by RenRick — DISMISSED** [review]
LGTM

**Review by Fynnss — DISMISSED** [review]

File: graphql/graphql_test.go [hunk 1]
```diff
@@ -452,6 +453,7 @@ func newGQLService(t *testing.T, stack *node.Node, shanghai bool, gspec *core.Ge
 		TrieDirtyCache: 5,
 		TrieTimeout:    60 * time.Minute,
 		SnapshotCache:  5,
+		StateScheme:    rawdb.HashScheme,
```
Comment (Fynnss) [inline graphql/graphql_test.go:456]: Does the archive mode is needed for GQL test? if not, why still use hashScheme?

Comment (sysvm) [reply graphql/graphql_test.go:456]: Yes, GQL test still uses hash scheme and this code is merged from ethereum.

File: Makefile [hunk 2]
```diff
@@ -31,7 +31,7 @@ truffle-test:
 	docker build . -f ./docker/Dockerfile.truffle -t truffle-test
 	docker-compose -f ./tests/truffle/docker-compose.yml up genesis
 	docker-compose -f ./tests/truffle/docker-compose.yml up -d bsc-rpc bsc-validator1
-	sleep 30
```
Comment (zzzckck) [inline Makefile:34]: why?

**Review by zzzckck — DISMISSED** [review]

File: Makefile (hunk 2 above)
Comment (sysvm) [reply Makefile:34]: In snap sync, PBSS should wait for all state fully synced. Therefore, sleep more time.

**Review by RenRick — APPROVED** [review]
LGTM

**Review by buddh0 — APPROVED** [review]

**Review by zzzckck — APPROVED** [review]

**Pull request merged**